docs/API_docs/UserInfoAPI.md

```diff
@@ -27,51 +27,81 @@ Possible return message:
 
 ```
 {
-    "_id": {
-        "$oid": "5e43132fe3fc7eea4a59b5a4"
-    },
-    "username": "Arianne",
+    "username": "Sebastien",
+    "bio": "Welcome to mypanda space!!",
+    "nb_pictures": 5,
     "pictures": [
         {
-            "_id": {
-                "$oid": "5e431352e3fc7eea4a59b5a5"
-            },
-            "date": {
-                "$date": 1581436162806
-            },
-            "owner": "Arianne",
-            "user": "Arianne",
-            "link": "picture 1",
-            "message": "Awesome",
+            "owner": "Sebastien",
+            "user": "Sebastien",
+            "link": "www.panda.space",
+            "message": "Yep this is me",
+            "nb_likes": 0,
+            "nb_comments": 4,
```

Comment:
dont need this field

Reply:
Maybe, but it could be use by frontend to display the number of comments of a picture. I suggest we keep it

Reply:
It's not being displayed anywhere in the ui so It's not very useful

Reply:
Well, you should display it